app/config.js

```diff
@@ -144,6 +144,18 @@ const config = convict({
     default: '',
     env: 'AWS_ROLE_ARN',
   },
+  ENABLE_MOCK_TIME: {
+    doc: 'Enable Mock Time',
+    format: Boolean,
+    default: false,
+    env: 'ENABLE_MOCK_TIME',
+  },
+  ENABLE_MOCK_COOKIES: {
+    doc: 'Enable DB Mocks Cookies',
+    format: Boolean,
+    default: false,
+    env: 'ENABLE_MOCK_COOKIES',
+  },
```

Comment:
We should ensure that these options are never set in the prod environment. See 
https://github.com/bcgov/cas-cif/blob/359d5dcacfd88874aa70aedf5550f32610c6750c/app/config.js#L220 for an example

Reply:
Implemented